docs/using/plugins.md

```diff
@@ -13,7 +13,7 @@ Currently, systems using custom interactions can only be exported to OpenMM. The
 
 ## Creating a custom `ParameterHandler`
 
-There is no formal specification for a `ParameterHandler` as processed by the OpenFF Toolkit, but there are some guidelines that are expected to be followed. A parameter handler requires both a [`ParameterHandler`] class, which performs parametrization, and a [`ParameterType`] class, which stores the corresponding data. For more information, see the [Toolkit documentation](openff.toolkit:users/developing.md#modular-design-features).
+There is no formal specification for a `ParameterHandler` as processed by the OpenFF Toolkit, but there are some guidelines that are expected to be followed. A parameter handler requires both a [`ParameterHandler`] class, which performs parametrization, and a [`ParameterType`] class, which stores the corresponding data. For more information, see the Toolkit documentation.
```

Comment:
@Yoshanuikabundi I had to drop this reference to get docs to build. Here's the warning which I believe cause the build to fail:

```
/home/docs/checkouts/readthedocs.org/user_builds/openff-interchange/checkouts/878/docs/using/plugins.md:16: WARNING: 'myst' reference target not found: openff.toolkit:users/developing.md#modular-design-features
```

Any idea what's going on here?

https://readthedocs.org/projects/openff-interchange/builds/23287319/